Framework/Algorithms/src/DetectorEfficiencyCor.cpp

```diff
@@ -245,9 +251,9 @@ void DetectorEfficiencyCor::correctForEfficiency(
     // now get the sin of the angle, it's the magnitude of the cross product of
```

Comment:
Looking at this again, I think this is actually computing a different angle, that is also theta, i.e., keep it as it is :+1: